app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
+# Helper methods
 module ApplicationHelper
-end
+  def login_helper (style)
```

Comment:
@sohilp393 , Do not put a space between a method name and the opening parenthesis.

Reply:
Removed the space between method name and opening parenthesis .ALso removed the rubocop warning.